src/screens/Platform/CatalogBanner.js

```diff
@@ -34,15 +36,25 @@ const styles = StyleSheet.create({
     borderRadius: 4,
     padding: 16,
     marginBottom: 16,
+    overflow: "hidden",
+    position: "relative",
   },
   title: {
     fontSize: 18,
-    marginBottom: 16,
+    marginBottom: 7,
```

Comment:
We _try_ to follow 2,4,8,12,16,32 for distance so this should be an 8 even if on figma it's a 7, but 🤷🏼‍♂️

Reply:
Yes that was the value on figma. I'll fix this right now ! 👍